project_task_stage_change_restriction/__manifest__.py

```
@@ -0,0 +1,24 @@
+# Copyright (C) 2025 Cetmix OÜ
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Project Task Stage Change Restriction",
+    "summary": "Restrict project task stage",
+    "version": "16.0.1.0.0",
+    "category": "Project",
```

Flake8 (7.2.0)

[error] dictionary key 'category' repeated with different values

(F601)